packaging/grafana-agent-flow/windows/install_script.nsis

```diff
@@ -111,6 +111,14 @@ Function CreateConfig
     Return
   CreateNewConfig:
     File "config.river"
+
+    # Set permissions on the config file
```

Comment:
(Discussed offline, but writing it here for transparency)

I like that this applies only when the config file is first created, since it allows users to have custom permissions and not worry about them getting overwritten. Right now, the permissions for INSTDIR get updated every time one of these installers is ran, and I think similar logic should apply; only apply permissions the very first time the folder is created.

Reply:
See the latest for this